<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Emotion_Train_ToolClass</class>
 <widget class="QMainWindow" name="Emotion_Train_ToolClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Emotion_Train_Tool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>531</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0" columnstretch="3,1">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="1" column="1">
      <widget class="QPushButton" name="choose_path">
       <property name="text">
        <string>Choose Path</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QListWidget" name="fileList"/>
     </item>
     <item row="1" column="0">
      <widget class="QSlider" name="video_status">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QScrollArea" name="scrollArea">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>400</height>
        </size>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustIgnored</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>398</width>
          <height>424</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="picshow">
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="but_capture">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>75</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Capture</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="group_emotion">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>490</y>
      <width>311</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Emotion Type:</string>
    </property>
    <widget class="QPushButton" name="but_happy">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Happy</string>
     </property>
    </widget>
    <widget class="QPushButton" name="but_normal">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Normal</string>
     </property>
    </widget>
    <widget class="QPushButton" name="but_sleepy">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Sleepy</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="emotion_train_tool.qrc"/>
 </resources>
 <connections/>
</ui>
